README.md

```diff
@@ -41,19 +41,9 @@ Please see our [iOS Integration Guide](https://stripe.com/docs/mobile/ios) which
 
 There are 3 example apps included in the repository:
 
-- **UI Examples**: Shows off the various premade UI components the SDK includes, but is not a functional app and makes no network calls.
-- **Standard Integration (Swift)**: shows an integration using our prebuilt UI components.
-- **Custom Integration (ObjC)**: shows how to use our low-level methods to accept payments using several different payment methods.
-
-To try out any of these examples, clone this repository, open `Stripe.xcworkspace` with Xcode, and run the appropriate scheme. The **UI Examples** app can be run without any extra work necessary. In order to run the **Standard Integration (Swift)** or **Custom Integration (ObjC)** example, you must perform these setup steps:
-
-1. Execute `./setup.sh` from the root of the repository to build the necessary dependencies.
-2. If you haven't already, sign up for a [Stripe account](https://dashboard.stripe.com/register) (it takes seconds). Then go to https://dashboard.stripe.com/account/apikeys.
-3. Replace the `stripePublishableKey` constant in CheckoutViewController.swift (for the Standard Integration app) or Constants.m (for the Custom Integration app) with your Test Publishable Key.
-4. Head to https://github.com/stripe/example-ios-backend/tree/v11.0.0 and click "Deploy to Heroku" (you may have to sign up for a Heroku account as part of this process). Provide your Stripe test secret key for the STRIPE_TEST_SECRET_KEY field under 'Env'. Click "Deploy for Free".
-5. Replace the `backendBaseURL` variable in the example iOS app with the app URL Heroku provides you with (e.g. "https://my-example-app.herokuapp.com")
-
-After this is done, you can make test payments through the app and see them in your Stripe dashboard. Head to https://stripe.com/docs/testing#cards for a list of test card numbers.
+- [UI Examples](/Example/UI%20Examples/README.md)
+- [Standard Integration (Swift)](/Example/Standard%20Integration%20%28Swift%29/README.md)
+- [Custom Integration (ObjC)](/Example/Custom%20Integration%20%28ObjC%29/README.md)
```

Comment:
I think it'd be better if this section had more details. When browsing via github, I doubt I would know that these led to the corresponding `README` files, and I'd probably miss out on the setup instructions.

<img width="580" alt="screen shot 2018-06-06 at 2 35 52 pm" src="https://user-images.githubusercontent.com/30532866/41066567-1ae18408-6997-11e8-8a80-96abfeefaf12.png">

Reply:
👍 how does this look?

<img width="703" alt="screen shot 2018-06-06 at 4 15 09 pm" src="https://user-images.githubusercontent.com/28276156/41069911-d083a7ac-69a4-11e8-9cdd-f0aef52204aa.png">

Reply:
👍 looks better